modules/ecs/cluster/env-vars-for-ecs.tf

```diff
@@ -0,0 +1,154 @@
+        }
+EOF
+
+  ecs_daw_shared_ssm_secrets = <<EOF
+        {
+          "name": "CLUSTER_NAME",
+          "valueFrom": "${aws_ssm_parameter.cluster_name.arn}"
+        },
+        {
+          "name": "AWS_REGION",
+          "valueFrom": "${aws_ssm_parameter.aws_region.arn}"
+        },
+        {
+          "name": "SES_PORT",
+          "valueFrom": "${aws_ssm_parameter.ses_port.arn}"
+        },
+        {
+          "name": "SES_SERVER",
+          "valueFrom": "${aws_ssm_parameter.ses_server.arn}"
+        },
+        {
+          "name": "SES_FROM_EMAIL",
+          "valueFrom": "${aws_ssm_parameter.ses_from_email.arn}"
+        },
+        {
+          "name": "SMTP_USERNAME",
+          "valueFrom": "${aws_ssm_parameter.smtp_username.arn}"
+        },
+        {
+          "name": "SMTP_PASSWORD",
+          "valueFrom": "${aws_ssm_parameter.smtp_password.arn}"
+        },
+        {
+            "name": "SECRET_KEY_BASE",
+            "valueFrom": "${aws_ssm_parameter.secret_key_base.arn}"
+        },
+        {
+            "name": "S3_SSL_BUCKET",
+            "valueFrom": "${aws_ssm_parameter.s3_ssl_bucket.arn}"
+        },
+        {
+            "name": "S3_LOG_BUCKET",
+            "valueFrom": "${aws_ssm_parameter.s3_log_bucket.arn}"
+        },
+        {
+            "name": "HOST",
+            "valueFrom": "${aws_ssm_parameter.host.arn}"
+        }
+EOF
+
+  ecs_da_shared_ssm_secrets = <<EOF
+        {
+            "name": "PORT",
+            "valueFrom": "${aws_ssm_parameter.port.arn}"
+        }
+EOF
+
+  ecs_daw_shared_ssm_secrets = <<EOF
+        {
+          "name": "CLUSTER_NAME",
+          "valueFrom": "${aws_ssm_parameter.cluster_name.arn}"
+        },
+        {
+          "name": "AWS_REGION",
+          "valueFrom": "${aws_ssm_parameter.aws_region.arn}"
+        },
+        {
+          "name": "SES_PORT",
+          "valueFrom": "${aws_ssm_parameter.ses_port.arn}"
+        },
+        {
+          "name": "SES_SERVER",
+          "valueFrom": "${aws_ssm_parameter.ses_server.arn}"
+        },
+        {
+          "name": "SES_FROM_EMAIL",
+          "valueFrom": "${aws_ssm_parameter.ses_from_email.arn}"
+        },
+        {
+          "name": "SMTP_USERNAME",
+          "valueFrom": "${aws_ssm_parameter.smtp_username.arn}"
+        },
+        {
+          "name": "SMTP_PASSWORD",
+          "valueFrom": "${aws_ssm_parameter.smtp_password.arn}"
+        },
+        {
+            "name": "SECRET_KEY_BASE",
+            "valueFrom": "${aws_ssm_parameter.secret_key_base.arn}"
+        }
+EOF
+
+  ecs_dac_shared_ssm_secrets = <<EOF
+        {
+            "name": "HOST",
+            "valueFrom": "${aws_ssm_parameter.host.arn}"
+        }
+EOF
+
+  ecs_da_shared_ssm_secrets = <<EOF
+        {
+            "name": "PORT",
+            "valueFrom": "${aws_ssm_parameter.port.arn}"
+        }
+EOF
+
+  ecs_shared_env_vars = <<EOF
+        { "name" : "MIX_ENV", "value" : "${var.profile}" },
```

Comment:
We might need some of these at some point but we're only passing in two env vars at the moment to the containers. APP_NAME and ENVIRONMENT.

Reply:
@magreenbaum okay so remove all but those two

Reply:
removed ✅